gradlew

```diff
@@ -76,14 +76,12 @@ if [ -n "$JAVA_HOME" ] ; then
     fi
```

Comment:
Please revert as it was before

Reply:
It hasn't been removed. It only swapped place (See line 9). I guess this has something to do with pulling the latest version of master.

Reply:
Your file should match what is in master I shouldn't see any changes on this file compared to master.